website/docs/r/license.html.markdown

```diff
@@ -32,15 +28,15 @@ resource "vsphere_license" "licenseKey" {
 
 The following arguments are supported:
 
-* `license_key` - (Required) The key of the license which is needed to be added to vpshere
-* `label` - (Optional) The key value pair of labels that has to be attached to the license key. One key can have multiple labels.
+* `license_key` - (Required) The license key to add.
+* `labels` - (Optional) The key value pair of labels that has to be attached to the license key. 
```

Comment:
Looking good here! Thank you for the fix!

Just a couple of remaining nitpicks on the `labels` line:
 * Can we indicate that it's a map now?
 * Can we say `key/value` instead of `key value`?
 * It says `has to be attached`, but it's optional.

Something like:

> `labels` - (Optional) A map of key/value pairs to be attached as labels (tags) to the license key.